snippets/large_set.js

```diff
@@ -4,16 +4,25 @@ yourDiv.textContent = "loading";
 
 var opts = {};
 opts.el = yourDiv;
+
+const customValueCalculator = (id, currentSequence) => {
+  return Math.random() * 100;
+};
+
 opts.vis = {
   conserv: false,
-  overviewbox: false
+  overviewbox: false,
+  labelCustomColumnsNames: ['homoscore'],
+  labelCustomColumnsValuesGetter: [customValueCalculator],
+  labelCustomColumns: true,
```

Comment:
Let's use a lazier API that also supports rendering HTML elements.

How about `customColumnsLength`, and `customColumnGetter`.
`customColumnGetter` will be called by MSA with column indices from `[0, customColumnsLength)`, and the client returns back information about the header, body cell values, and width of the column.